tools/docker/old-env/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q --no-install-recommends
 	apt-transport-https software-properties-common g++ \
 	g++-arm-linux-gnueabi g++-aarch64-linux-gnu g++-powerpc64le-linux-gnu \
 	g++-mips64el-linux-gnuabi64 g++-s390x-linux-gnu \
-	linux-libc-dev:i386 lib32gcc-5-dev lib32stdc++-5-dev \
+	linux-libc-dev:i386 lib32gcc-5-dev lib32stdc++-5-dev zlib1g zlib1g-dev \
```

Comment:
Just in case -- shouldn't it also be `:i386` here?

Reply:
Uh, we also need arm64 and riscv versions. Not sure if they even exist on x86...
We cross-compile syz-executor and C repros to these arches on syzbot.

Reply:
I've tried to update the `Dockerfile`s to include the various versions we need - but there are a couple of issues:
- there doesn't seem to be a `riscv64` version of `zlib1g-dev` in official sources (https://packages.debian.org/bullseye/zlib1g-dev)
- `old-env` didn't seem to find the right repositories for other architectures except `i386`

What would you suggest the best approach is here? We could potentially cross-compile `zlib` from source, but I don't know if there is a better solution.